bucket/winscan2pdf.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "5.15",
+    "description": "Tiny utility to scan documents directly to PDF using any scanner",
+    "homepage": "http://www.softwareok.com/?seite=Microsoft/WinScan2PDF",
```

Comment:
```suggestion
    "homepage": "https://www.softwareok.com/?seite=Microsoft/WinScan2PDF",
```

Reply:
Good point, @linsui ! I just fixed this pull and also pull #3456. Thanks.